src/ensembl/src/content/app/entity-viewer/EntityViewer.tsx

```diff
@@ -71,6 +71,7 @@ const EntityViewer = (props: Props) => {
             sidebarNavigation={<GeneViewSidebarTabs />}
             sidebarToolstripContent={<EntityViewerSidebarToolstrip />}
             topbarContent={<EntityViewerTopbar />}
+            isSidebarFloating={true}
```

Comment:
"floating" is the wrong word here. `sidebarSlideOver: boolean`? or `sidebarBehaviour: 'push' | 'slideover'`?

Reply:
How about `sidebarSide` instead of `sidebarBehaviour`?

Reply:
✔️